brownie/network/event.py

```diff
@@ -172,15 +177,15 @@ def get_topics(abi):
     return eth_event.get_topics(abi)
 
 
-def decode_logs(logs):
+def decode_logs(logs: Optional[str]) -> Union['EventDict', List[None]]:
```

Comment:
`logs` will be a `List`..  I don't think it's optional either, the check for truthyness is just to see if the list is empty.